website/static/js/fangorn.js

```diff
@@ -1698,6 +1816,65 @@ var FGToolbar = {
                     }, 'Delete Multiple')
                 );
             }
+            if (showCheckout){
+                generalButtons.push(
+                    m.component(FGButton, {
+                        onclick: function() {
+                            items.forEach(function(each) {
+                                $.ajax({
```

Comment:
This call is very similar to the other ajax call for checking in a file. Can you refactor this into a helper function?